examples/disruption_crons/disruption_cron_network_drop.yaml

```diff
@@ -9,7 +9,7 @@ metadata:
   name: network-drop
   namespace: chaos-demo
 spec:
-  schedule: "*/15 * * * *"
```

Comment:
why that change?

Reply:
Ah I was using this for testing, going to revert to 15, thanks.